commands/uploader.go

```diff
@@ -31,7 +31,9 @@ func uploadForRefUpdates(ctx *uploadContext, updates []*refUpdate, pushAll bool)
 
 	verifyLocksForUpdates(ctx.lockVerifier, updates)
 	for _, update := range updates {
-		q := ctx.NewQueue() // initialized here to prevent looped defer
+		q := ctx.NewQueue( // initialized here to prevent looped defer
```

Comment:
Nit: should this comment be moved to the line above? I think if we ever add arguments to this method, it will help keep the lines short.

Reply:
Yeah I suppose. The comment could be moved when the first comment is added.